media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -165,7 +166,16 @@ src_compile() {
 	cat > ${limitsdfile} <<- EOF || die
 		# Start of ${limitsdfile} from ${P}
 
-		1000:60000	-	memlock 256
+		# Required to memlock buffers for pipewire-pulse & JACK emulation
+		@pipewire-desktop	-	memlock	256
+
+		# May be lowered/removed in the future; PW runs at 20 via rtkit
+		#@pipewire-desktop	-	rtprio	19
+
+		# Should match Red Hat defaults for their realtime group
```

Comment:
No, we will not overwrite settings for realtime group in this package. `USE=jack-sdk` will probably pull in `sys-auth/realtime-base`.

Reply:
Because we install 00-foo.conf while the realtime-base installs 40-foo.conf, our @realtime settings will be replaced by that file's higher values - the intention is to provide slightly less insane settings for users who do not already have a pro-audio setups (frankly I don't like rtprio 70 either but as you discovered at least it's what Red Hat installs and it's not 99). With further testing I hope we can settle on no more than rtprio 49 but first we need JACK applications that attempt to schedule themselves at FIFO or RR priority levels in the first place.